application/views/catalog/partials/advanced_search.php

```diff
@@ -100,10 +100,9 @@
 	              </button>
 	         </div>
 	       </div>
-
-               <h5>Still can't find what you're looking for? Try searching <a href="https://archive.org/details/librivoxaudio">Internet Archive's LibriVox Audiobook Collection</a>
-                 by title, author, reader, key terms, or terms appearing in book descriptions.
-               </h5>
+		
+		<h4>Couldn't find what you're looking for?</h4>
+		<h5>You can also try searching for Librivox audiobooks by title, author, reader, key terms, or terms appearing in book descriptions at <A href="https://archive.org/details/librivoxaudio?tab=collection" target="_blank">Internet Archive Librivox Free Audiobook Collection.</A> </h5>
```

Comment:
Besides my disapproval of the _blank target (see https://github.com/LibriVox/librivox-catalog/pull/203#discussion_r1518617340) and the use of the capital A for the <a> elements, I don't really have an opinion on whether this is an improvement over https://github.com/LibriVox/librivox-catalog/pull/203#discussion_r1518617340. @redrun45 cowardly leaving it up to you to make the call.

Reply:
The initial one wasn't so much 'my call', as I was the one who happened to do it.  I'll see if other admins have an opinion.  Posting screenshots, and we'll go from there.

Reply:
I don't know what the preferences are for keeping commit history "clean", but at any rate, [here's](https://github.com/LibriVox/librivox-catalog/pull/206) the version that admins we'd like to go with.